packages/core/src/App/Components/Elements/live-chat.jsx

```diff
@@ -1,22 +1,101 @@
 import React from 'react';
 import Cookies from 'js-cookie';
+import { useHistory } from 'react-router-dom';
 import { connect } from 'Stores/connect';
 import { Popover, Icon } from '@deriv/components';
 import { localize } from '@deriv/translations';
+import { livechat_license_id } from '@deriv/shared';
 
 const LiveChat = ({ is_mobile_drawer, has_cookie_account }) => {
+    // Initialization code provided by Livechat
+    const liveChatInitialization = () =>
```

Comment:
nitpick: maybe we can move the portion of the `LiveChat` initialisation code to another file?

Reply:
Thought of the same thing too...but I wasn't sure where would be the best place to put it...any suggestions? Shall i create one file in core/src/utils?

Reply:
We can move the `live-chat.jsx` into a folder e.g - `LiveChat` and have both files there respectively as `live-chat.jsx` and `live-chat.js`. 🙏🏻